numpy/ctypeslib.py

```diff
@@ -527,8 +568,52 @@ def as_array(obj, shape=None):
 
 
     def as_ctypes(obj):
-        """Create and return a ctypes object from a numpy array.  Actually
-        anything that exposes the __array_interface__ is accepted."""
+        """
+            as_ctypes(obj)
+
+            Create and return a ctypes object from a numpy array.  Actually
```

Comment:
Maybe say ``ctypes array`` instead of ``ctypes object``.  Then we do not need the last sentence anymore either?  (It does not convert dtypes, but arrays – so the contained data.)